src/vaadin-time-picker.html

```diff
@@ -290,6 +293,7 @@
 
             /**
              * Set true to prevent the overlay from opening automatically.
+             * @type {boolean}
```

Comment:
Same as above, `@type` annotation can be removed so the default `boolean | null | undefined` would apply.

Reply:
Done